vcd/resource_vcd_vapp_vm.go

```diff
@@ -563,63 +557,58 @@ func resourceVcdVAppVmUpdate(d *schema.ResourceData, meta interface{}) error {
 		}
```

Comment:
Can't comment on the `resourceVcdVAppVmUpdate` signature, but it needs a lock. I was able to get such error: `Error: error Powering Off: &errors.errorString{s:"error powering off VM: API Error: 400: [ 4001d52e-3028-4474-921e-215f8a201348 ] The entity web1 is busy completing an operation VAPP_POWER_OFF. VAPP_POWER_OFF(com.vmware.vcloud.entity.task:1a66a737-99bc-4a0c-89e7-572e4f107069)"}`

Reply:
ok. I cannot add directly cause it will create deadlock - will create wrapper.

Reply:
Added

Reply:
The same test I had failing works fine now 👍